types/src/app.d.ts

```diff
@@ -37,3 +37,6 @@ export type FilterNavButton = {
   label: string; // label to display in the button
   action?: (payload: MouseEvent) => void; // action to take when we click the button
 };
+
+// type of entities on Etherscan.
+export type EtherscanGroup = 'tx' | 'token' | 'address';
```

Comment:
Per the guidance in https://github.com/dcgtc/dgrants/discussions/114, this type definition should live in the `app` package instead of the `types` package. In your defense @carletex, it looks like we haven't been following that guidance well haha

Reply:
haha ok! Didn't know about that guide. Thanks!